drivers/bluetooth/hci/userchan.c

```diff
@@ -289,9 +327,23 @@ static void add_btuserchan_arg(void)
 
 static void btuserchan_check_arg(void)
 {
-	if (bt_dev_index < 0) {
-		posix_print_error_and_exit("Error: Bluetooth device missing. "
-					   "Specify one using --bt-dev=hciN\n");
+	if (!arg_found) {
+		posix_print_error_and_exit("Error: Bluetooth device missing.\n"
+					   "Specify either a local hci interface --bt-dev=hciN\n"
+					   "or a valid hci tcp server --bt-dev=ip_address:<port>\n");
```

Comment:
very minor: Should it be either `<ip_address>:<port>` or `ip_address:port`? (maybe the second better for consistency with other errors?)